src/github-issues.ts

```diff
@@ -132,3 +133,49 @@ export async function removeEyes(
     }
   }
 }
+
+/**
+ * Get detailed information about an issue including comments
+ */
+export async function getIssueDetails(
+  octokit: ReturnType<typeof github.getOctokit>,
+  owner: string,
+  repo: string,
+  issueNumber: number
+): Promise<IssueDetails> {
+  const { data: issue } = await octokit.rest.issues.get({
+    owner,
+    repo,
+    issue_number: issueNumber
+  })
+
+  // Get issue comments
+  const { data: comments } = await octokit.rest.issues.listComments({
+    owner,
+    repo,
+    issue_number: issueNumber
+  })
+
+  const commentsData: CommentData[] = comments.map((comment) => ({
+    id: comment.id,
+    user: comment.user!,
+    created_at: comment.created_at,
+    reactions: comment.reactions!
+  }))
+
+  return {
+    id: issue.id.toString(),
+    number: issue.number,
+    title: issue.title,
+    body: issue.body || '',
+    state: issue.state,
+    created_at: issue.created_at,
+    updated_at: issue.updated_at,
+    closed_at: issue.closed_at,
+    comments: issue.comments,
+    reactions: issue.reactions!,
```

Comment:
We need to get the reactions details with the created date. Update all the types and historical functions and everything to use the date. 

Have a look at this file and I think switch to the GraphQL version. Please paginate. There may be many issues, comments, reactions, etc: https://github.com/mattleibow/dev-tools/blob/main/LabeledByAI.Services/GitHub/GitHubRemoteConnection.cs